0xProject/0x-monorepo
# [instant] [asset-buyer] Helper methods for checking liquidity

## Description

Adds `getLiquidityForAssetDataAsync` function to `AssetBuyer` class.

Adds following helper functions to `zeroExInstant`:
- `assetDataForERC20TokenAddress`
- `hasMetaDataForAssetData`
- `hasLiquidityForAssetDataAsync`

Adds mocking abilities to AssetBuyer using `typemoq`


⚠️ After Merge ⚠️ 

- Update wiki docs to include these new functions
- Respond to integrator who requested this type of functionality


## Testing instructions

See demo at http://0x-instant-dogfood.s3-website-us-east-1.amazonaws.com/liquidity_demo.html (can view source for code)


## Types of changes









## Checklist:
